### PR TITLE
WIP: use react

### PR DESCRIPTION
**Pull request by myakove**

### CODE REVIEW

**Comment by redhat-qe-bot1** [comment]
Report bugs in [Issues](https://github.com/myakove/github-webhook-server/issues)

The following are automatically added:
 * Add reviewers from OWNER file (in the root of the repository) under reviewers section.
 * Set PR size label.
 * New issue is created for the PR. (Closed when PR is merged/closed)
 * Run [pre-commit](https://pre-commit.ci/) if `.pre-commit-config.yaml` exists in the repo.

Available user actions:
 * To mark PR as WIP comment `/wip` to the PR, To remove it from the PR comment `/wip cancel` to the PR.
 * To block merging of PR comment `/hold`, To un-block merging of PR comment `/hold cancel`.
 * To mark PR as verified comment `/verified` to the PR, to un-verify comment `/verified cancel` to the PR.
        verified label removed on each new commit push.
 * To cherry pick a merged PR comment `/cherry-pick <target branch to cherry-pick to>` in the PR.
    * Multiple target branches can be cherry-picked, separated by spaces. (`/cherry-pick branch1 branch2`)
    * Cherry-pick will be started when PR is merged
 * To build and push container image command `/build-and-push-container` in the PR (tag will be the PR number).
 * To add a label by comment use `/<label name>`, to remove, use `/<label name> cancel`
 * To assign reviewers based on OWNERS file use `/assign-reviewers`
 * To check if PR can be merged use `/check-can-merge`

<details>
<summary>Supported /retest check runs</summary>

 * `/retest build-container`: Retest build-container

</details>

<details>
<summary>Supported labels</summary>

 * hold
 * verified
 * wip
 * lgtm

</details>

File: api/utils.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            "namespace": _namespace or "Not Ready",
+            "pool": _instnce.spec.clusterPoolName,
+        }
+        _cluster_info["info"] = []
+        if _namespace:
+            _info_dict = {
+                "console": get_claimed_cluster_web_console(claim_name=_name),
+                "kubeconfig": get_claimed_cluster_kubeconfig(claim_name=_name),
+                "creds": get_claimed_cluster_creds(claim_name=_name),
+                "name": _instnce.metadata.name,
```
Comment (rnetser) [inline api/utils.py:33]:
```suggestion
                "name": _name,
```

File: api/utils.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    return {"deleted_claims": deleted_claims}
+
+
+def get_claimed_cluster_deployment(claim_name: str) -> ClusterDeployment | str:
+    _claim: Any = ClusterClaim(name=claim_name, namespace=HIVE_CLUSTER_NAMESPACE)
+    _instance: ResourceInstance = _claim.instance
+    if not _instance.spec.namespace:
+        return "<p><b>ClusterDeployment not found for this claim</b></p>"
+
+    return ClusterDeployment(name=_instance.spec.namespace, namespace=_instance.spec.namespace)
```
Comment (rnetser) [inline api/utils.py:120]: `_instance.spec` save

**Comment by myakove** [comment]
Moved to 
https://github.com/RedHatQE/hive-claim-manager-server
https://github.com/RedHatQE/hive-claim-manager-client

**Pull request closed without merging**